client/babel.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  presets: [
+    '@vue/cli-plugin-babel/preset'
+  ]
+}
```

Comment:
This is completely different from ours (https://github.com/bcgov/lear/blob/master/coops-ui/babel.config.js) but that may be because we enabled the "Progressive Web App" option and it looks like you didn't. I don't have a solid opinion on this.

Reply:
I just let @vue/cli-service handle that for me.  The tools it uses have changed quite a bit in the last while.  I presume it should work out of the box (hopefully)